packages/nx/src/command-line/add/add.ts

```diff
@@ -175,6 +176,21 @@ function parsePackageSpecifier(
   return [pkgName, version];
 }
 
+const knownCommandArgs = [
+  'packageSpecifier',
+  'updatePackageScripts',
+  'verbose',
+  '__overrides_unparsed__',
+  '$0',
+];
+function getExtraArgs(options: AddOptions): string[] {
+  const unknownOptions = Object.keys(options)
```

Comment:
I believe if you set the yargs parser configuration like here then you won't see other options in `options`, only `__unparsed_options__`

https://github.com/nrwl/nx/blob/322ecf7a088eb195da289674224eccb9f3320075/packages/nx/src/command-line/yargs-utils/shared-options.ts#L141

Reply:
Nice! TIL